### ISSUE TITLE
Needs to exclude `aws-sdk`

### ISSUE THREAD

**Issue by johnhaley81** [e-e-e/serverless-plugin-inject-dependencies#3]
Currently, `aws-sdk` will always be bundled. This is a special case and should be skipped as an included module.

**Comment by e-e-e** [comment]
Yeah - I have noticed this but did not have time to jump on it yet. Thanks for raising the issue.
